bessctl/bessctl

```diff
@@ -1,24 +1,25 @@
 #!/usr/bin/env python2.7
```

Comment:
Now all scripts should start with "#!/usr/bin/env python", so that they can run regardless of the installed Python version..?